src/components/employees/employeeCardGroup.tsx

```diff
@@ -116,3 +142,48 @@ export const sortEmployeesData = (data: Employee[]) => {
 
   return sortedData;
 };
+
+// I'm going to be honest, this isn't the way I wanted to calculate it, but I couldn't find a better way. Feel free to improve it.
+const CheckScreenWidth = (data: {
```

Comment:
What are you trying to achieve here? Can you explain why and how this is setup? :D (in slack is fine)